app/code/Magento/QuoteGraphQl/README.md

```diff
@@ -0,0 +1,4 @@
+# QuoteGraphQl
+
+**QuoteGraphQl** provides type and resolver information for the GraphQl module
```

Comment:
In GraphQL we should not use word `endpoint`, instead we have queries and mutations